### PR TITLE
Replace links to support email with support website

### LINKED ISSUE #378: Update support@stoplight.io references within documentation
## Context

On June 6th, 2022 support@stoplight.io will no longer create tickets for users. Can you please replace all references of support@stoplight.io with https://support.stoplight.io ?  

Below are some of the references I have found, but this is not an exhaustive list. 
- https://meta.stoplight.io/docs/platform/ZG9jOjE0OTg5MjQ1-approved-email-domains 
- https://meta.stoplight.io/docs/platform/3aaeb73f741fc-faq
- https://meta.stoplight.io/docs/platform/ZG9jOjI4MDczOTQ0-billing

### PR DESCRIPTION
**Pull request by heitortsergent**
Closes #378.

### CODE REVIEW

File: 6.-migrations/v1.md [hunk 1]
```diff
@@ -71,7 +72,8 @@ The fastest way to get your team moved over to Stoplight Platform is to [configu
 
 Learn about the different workpsace roles [here](../2.-workspaces/k.workspace-roles.md).
 
-> If you are using an enterprise git installation and need to configure your firewall to allow the connection, please don't hesitate to [reach out](mailto:support@stoplight.io) and we will schedule a time to assist you. 
+<!-- markdown-link-check-disable-next-line -->
+> If you are using an enterprise git installation and need to configure your firewall to allow the connection, please don't hesitate to [reach out](https://support.stoplight.io) and we will schedule a time to assist you.
```
Comment (pamgoodrich) [inline 6.-migrations/v1.md:76]: git > Git

Comment (pamgoodrich) [reply 6.-migrations/v1.md:76]: Remove please

File: 6.-migrations/v1.md [hunk 2]
```diff
@@ -85,8 +87,10 @@ V2 comes with up to date hosted mock servers to help speed up development. Follo
 
 ### Setup Design Libraries
 
+<!-- markdown-link-check-disable -->
 <!-- theme: warning -->
-> Design libraries in V2 are not a public feature and are limited to existing V1 users only. If you are a V1 user and don't have design libraries enabled please don't hesitate to [reach out](mailto:support@stoplight.io) and we will assist you. 
+> Design libraries in V2 are not a public feature and are limited to existing V1 users only. If you are a V1 user and don't have design libraries enabled please don't hesitate to [reach out](https://support.stoplight.io) and we will assist you. 
+<!-- markdown-link-check-enable -->
 
 V2 comes with support for multiple design libraries. You can set complete git projects as design libraries unlike in V1 where you'd have to add schemas individually to a single design library.
```
Comment (pamgoodrich) [inline 6.-migrations/v1.md:95]: git >Git

File: 6.-migrations/v1.md [hunk 3]
```diff
@@ -96,7 +100,8 @@ To set a project as a design library, go to the project settings and designate t
 
 Next up go to the projects using design libraries, and update the links to point to the latest links. i.e. `https://stoplight.io/api/v1/projects/{workspace-name}/{project-name}/nodes/{node-url}`
 
-> If you are a heavy user of design libraries, please don't hesitate to [reach out](mailto:support@stoplight.io) and we will schedule time to assist with the migration. 
+<!-- markdown-link-check-disable-next-line -->
+> If you are a heavy user of design libraries, please don't hesitate to [reach out](https://support.stoplight.io) and we will schedule time to assist with the migration.
```
Comment (pamgoodrich) [inline 6.-migrations/v1.md:104]: Remove please

File: c.troubleshooting.md [hunk 4]
```diff
@@ -14,9 +15,8 @@ Try these solutions:
 2. Make sure the API is running and available under the specified URL.
 3. If you've checked all of the above and still experiencing issues, check if the API supports [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS). Requests from browsers will be blocked for APIs that do not have appropriate CORS headers set up for `*.stoplight.io.` 
 
-<!-- markdown-link-check-disable -->
+<!-- markdown-link-check-disable-next-line -->
 See our [Knowledge Base](https://support.stoplight.io/hc/en-us/articles/360046967991-Error-while-making-the-request-Network-Error-the-API-did-not-return-a-response) for guidance on resolving this error.
```
Comment (pamgoodrich) [inline c.troubleshooting.md:19]: @wiggins-chris Will this KB link need to change?

File: c.troubleshooting.md [hunk 5]
```diff
@@ -97,10 +98,12 @@ We aim to support the following browsers and minimum versions.
 
 Things may work outside of these browsers and these versions, but there might be unexpected problems as we won't be testing those environments.
 
-If something is not working in a version listed here (or newer) please [contact support](mailto:support@stoplight.io).
+<!-- markdown-link-check-disable-next-line -->
+If something is not working in a version listed here (or newer) please [contact support](https://support.stoplight.io).
```
Comment (pamgoodrich) [inline c.troubleshooting.md:102]: Remove please

File: c.troubleshooting.md [hunk 6]
```diff
@@ -97,10 +98,12 @@ We aim to support the following browsers and minimum versions.
...
 
 ## What languages are supported?
 
 Today Stoplight is only available in the English language. While it is possible to host your own content in Stoplight that is not English, you may experience minor errors and other discrepancies when using non-Latin letters or scripts (ie, Cyrillic-based characters). For example, only the Latin alphabet is supported in Stoplight's search functionality.
 
-For any questions or concerns, please [contact Stoplight Support](mailto:support@stoplight.io).
+<!-- markdown-link-check-disable-next-line -->
+For any questions or concerns, please [contact Stoplight Support](https://support.stoplight.io).
```
Comment (pamgoodrich) [inline c.troubleshooting.md:109]: Remove please. Honestly, I'd probably remove the whole sentence here, but that's up to Chris.

File: c.troubleshooting.md (hunk 4 above)
Comment (wiggins-chris) [reply c.troubleshooting.md:19]: good catch @pamgoodrich! Yes, here is the new link https://stoplighthelp.force.com/s/article/Error-while-making-the-request-Network-Error-the-API-did-not-return-a-response

File: c.troubleshooting.md (hunk 6 above)
Comment (wiggins-chris) [reply c.troubleshooting.md:106]: I agree with Pam, go ahead and remove it.

File: 6.-migrations/classic.md [hunk 7]
```diff
@@ -66,10 +66,8 @@ we'll notify you as each is released.
 
 ## FAQ
 
-The concepts in the new Stoplight Platform should be very similar to what you're
-used to in Classic. Here are some commonly asked questions to help clarify those
-similarities and differences. If you have any additional questions, please don't
-hesitate to [contact us](mailto:support@stoplight.io).
+<!-- markdown-link-check-disable-next-line -->
```
Comment (pamgoodrich) [inline 6.-migrations/classic.md:69]: Remove the word "please."

File: 6.-migrations/next.md [hunk 8]
```diff
@@ -84,7 +84,8 @@ Each feature is linked to a card on our public roadmap, and we encourage you to
 
 ## FAQ
 
-The concepts in the new Stoplight Platform should be very similar to what you're used to in NEXT. Here are some commonly asked questions to help clarify those similarities and differences. If you have any additional questions, please don't hesitate to [contact us](mailto:support@stoplight.io).
+<!-- markdown-link-check-disable-next-line -->
+The concepts in the new Stoplight Platform should be very similar to what you're used to in NEXT. Here are some commonly asked questions to help clarify those similarities and differences. If you have any additional questions, please don't hesitate to [contact us](https://support.stoplight.io).
```
Comment (pamgoodrich) [inline 6.-migrations/next.md:88]: Remove the word "please"

File: 6.-migrations/v1.md [hunk 9]
```diff
@@ -85,21 +84,19 @@ V2 comes with up to date hosted mock servers to help speed up development. Follo
 
 ### Setup Design Libraries
```
Comment (pamgoodrich) [inline 6.-migrations/v1.md:85]: I just that Setup is one word in the title. It should be "Set Up"

**Review by pamgoodrich — APPROVED** [review]

**Pull request merged**